static/js/zform.js

```diff
@@ -0,0 +1,168 @@
+var zform = (function () {
+
+var exports = {};
+
+function div(opts) {
+    return $('<div>').append(opts.contents);
+}
+
+function ul(opts) {
+    var elem = $('<ul>');
+    elem.append(opts.contents);
+    elem.css('padding', '3px');
+    return elem;
+}
+
+function li(opts) {
+    var elem = $('<li>');
+    elem.append(opts.contents);
+    elem.css('list-style', 'none');
+    elem.css('padding', '2px');
+    return elem;
+}
```

Comment:
Why not use a handlebars template for this stuff?  It's really painful to security-audit this kind of JS construction.

Reply:
(And also not super fun for designer types working on styling them)